### PR TITLE
Move more code into translation tools

### PR DESCRIPTION
**Pull request by mkoistinen**
Use translation tools more

### CODE REVIEW

File: aldryn_newsblog/tests/test_sitemaps.py [hunk 1]
```diff
@@ -80,12 +80,6 @@ def test_languages_support(self):
         with override('de'):
             de_article = self.create_article()
 
-        multilingual_sitemap = NewsBlogSitemap()
-        self.assertArticlesIn(
-            [multilanguage_article, en_article, de_article],
-            multilingual_sitemap
-        )
-
```
Comment (cyberstar) [inline aldryn_newsblog/tests/test_sitemaps.py]: What's wrong with this test?
if logic isn't changed maybe lets keep it?

**Comment by cyberstar** [comment]
LGTM :+1:

File: aldryn_newsblog/tests/test_sitemaps.py (hunk 1 above)
Comment (mkoistinen) [inline aldryn_newsblog/tests/test_sitemaps.py]: I think I was just thinking that this I18NSitemap is now in translation-tools and is tested there. We can keep it here too though.

Comment (mkoistinen) [inline aldryn_newsblog/tests/test_sitemaps.py]: The test is just rewritten.

**Pull request merged**